gratipay/models/participant.py

```diff
@@ -416,11 +417,11 @@ def add_email(self, email, resend_threshold='3 minutes'):
             nonce = self.db.one("""
                 UPDATE emails
                    SET verification_start=%s
-                 WHERE participant=%s
+                 WHERE participant_id=%s
                    AND address=%s
                    AND verified IS NULL
              RETURNING nonce
-            """, (verification_start, self.username, email))
+            """, (verification_start, self.id, email))
             if not nonce:
                 return self.add_email(email)
 
```

Comment:
@rohitpaulk tests are failing. :disappointed: All the email tests get suck in this recursive call. Do you have any idea why? I have dropped the `participant` column and modified email writes to only include `participant_id`